tests/conftest.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+
+import numpy as np
+import pandas as pd
+import ssms
+
+import hssm
+
+
+@pytest.fixture(scope="module")
+def data_ddm():
+    v_true, a_true, z_true, t_true = [0.5, 1.5, 0.5, 0.5]
```

Comment:
Just a note: Might be good to test a range of parameter sets here for robustness in some cases. Those parameters are hard-coded here, and we might miss some strange things with the likelihoods if we fixate only on this (and the one below) parameter set.